doc/MAPGEN.md

```diff
@@ -223,8 +223,9 @@ apartments_mod_tower overmap terrain ids specified.
 ### Define mapgen "weight"
 
 (optional) When the game randomly picks mapgen functions, each function's weight value determines how rare it is. 1000
-is the default, so adding something with weight '500' will make it appear 1/3 times, unless more functions are added.
-(An insanely high value like 10000000 is useful for testing)
+is the default, so adding something with weight '500' will make it appear about half as often as others ("about"
+because, for instance, 1000/3000 is 1/3, but 500/2500 is 1/5 - not 1/6). (An insanely high value like 10000000 is
+useful for testing.)
```

Comment:
I think you want to say something like "will make it appear about half as often as other alternatives with the default weight of 1000".  Don't try to make any statements about how often it appears relative to the same thing with a different weight; that's liable to be just confusing.

Reply:
I'm a perfectionist; you're right.

Reply:
```suggestion
is the default, so adding something with weight '500' will make it appear about half as often as others using the default weight. (An insanely high value like 10000000 is useful for testing.)
```